Update MsgResTonpy.trans

todd-p opened the pull request:
'that show DURING noon' - I'm not sure when the show is on. It is on right at 12pm or in the afternoon in general?  If so, we could change this to "at noon" or "in the afternoon".

What would you think of calling it the "Chibi-Sifter"?  'Sieve' and 'sifter' are synonyms, but I hear people say "sift" more often.

todd-p committed:
Update MsgResTonpy.trans

'that show DURING noon' - I'm not sure when the show is on. It is on right at 12pm or in the afternoon in general?  If so, we could change this to "at noon" or "in the afternoon".

What would you think of calling it the "Chibi-Sifter"?  'Sieve' and 'sifter' are synonyms, but I hear people say "sift" more often.

cjuub commented:
How did I not notice that "Do" when wanting to save? Haha, thanks!

About sieve vs sifter. I think we should keep it as sieve, I actually thought quite a bit of this when I began translating. They're synonyms, but sieve got strengthened by the fact that google image search showed more relevant images with sieve than sifter. Also, the Wikipedia article uses Sieve as the main word.

Another important point is that many images are already translated to sieve, changing would be too time consuming since we're not really correcting anything with sifter.

Fun fact: the developers call it "Saucer" in the game filenames, goes to show how bad their English is over there. :)

cjuub commented:
Oh right, since you're new to git, remove instances of sifter and update the pull request according to:
http://stackoverflow.com/questions/16748115/how-to-modify-github-pull-request

In the future, try splitting different edits into different commits. For example in this pull request: have one or more commits for the fixed obvious errors, and one commit for the sieve->sifter. This allows me to cherry-pick the commits I agree with. :)

todd-p committed:
Update MsgResTonpy.trans

Good argument for 'sieve'. Change back to 'sieve'.  Also changed "sieveing" to "sieving" whose spelling I see on Wikipedia.
